drivers/storage/rackspace/storage/rackspace_storage.go

```diff
@@ -0,0 +1,946 @@
+// /*
```

Comment:
Hi @akiraw,

I appreciate the package comment, but package comments must follow a [documented](https://blog.golang.org/godoc-documenting-go-code) standard. Please fix. Also, there is no need to double-comment with line comments protecting a block comment. Thanks!